server.js

```diff
@@ -570,8 +570,8 @@ app.post('/submitanswer', function(req, res) {
         req.session.user,
         req.body.answer,
         function(err, value) {
-            var result = value ? 'correct' : 'incorrect';
-            var status = value ? 200 : 500;
+            var result = value.correct ? value : 'incorrect';
```

Comment:
`var` -> `const`

Reply:
No difference really. If this was a huge function, then I would agree with you and change it right away. But its just a 2 lines function.